openslides_backend/action/actions/user/set_present.py

```diff
@@ -56,3 +59,33 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
                     is_present.remove(meeting_id)
                     instance["is_present_in_meeting_ids"] = is_present
                     yield instance
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        meeting = self.datastore.get(
+            FullQualifiedId(Collection("meeting"), instance["meeting_id"]),
+            ["committee_id", "users_allow_self_set_present"],
+        )
+        if has_organisation_management_level(
+            self.datastore, self.user_id, OrganisationManagementLevel.CAN_MANAGE_USERS
+        ):
+            return
+        elif meeting.get("committee_id") and has_committee_management_level(
```

Comment:
```suggestion
        elif has_committee_management_level(
```
a meeting always has a committee

Reply:
Removed.